src/common/utils/__tests__/PGTime.test.ts

```diff
@@ -160,4 +209,404 @@ describe('PGTime', function () {
+            strictEqual(testTime.minute, 15);
+          });
+          it('Should set the second', function () {
+            strictEqual(testTime.second, 30);
+          });
+          it('should default the millisecond to 0', function () {
+            strictEqual(testTime.millisecond, 0);
+          });
+        });
+        context('With just hour and minute and meridian', function () {
+          beforeEach(function () {
+            testTime = PGTime.fromDisplay('11:50 PM');
+          });
+          it('Should parse the meridian', function () {
+            strictEqual(testTime.meridian, MERIDIAN.PM);
+          });
+          it('Should set the hour', function () {
+            strictEqual(testTime.hour, 23);
+          });
+          it('Should set the minute', function () {
+            strictEqual(testTime.minute, 50);
+          });
+          it('Should default the second to 0', function () {
+            strictEqual(testTime.second, 0);
+          });
+          it('should default the millisecond to 0', function () {
+            strictEqual(testTime.millisecond, 0);
+          });
+        });
+        context('With just hour and minute', function () {
+          beforeEach(function () {
+            testTime = PGTime.fromDisplay('11:50');
+          });
+          it('Should assume meridian is AM', function () {
+            strictEqual(testTime.meridian, MERIDIAN.AM);
+          });
+          it('Should set the hour', function () {
+            strictEqual(testTime.hour, 11);
+          });
+          it('Should set the minute', function () {
+            strictEqual(testTime.minute, 50);
+          });
+          it('Should default the second to 0', function () {
+            strictEqual(testTime.second, 0);
+          });
+          it('should default the millisecond to 0', function () {
+            strictEqual(testTime.millisecond, 0);
+          });
+        });
+        context('With just hour and meridian', function () {
+          beforeEach(function () {
+            testTime = PGTime.fromDisplay('4 PM');
+          });
+          it('Should parse the meridian', function () {
+            strictEqual(testTime.meridian, MERIDIAN.PM);
+          });
+          it('Should set the hour', function () {
+            strictEqual(testTime.hour, 16);
+          });
+          it('Should default the minute to 0', function () {
+            strictEqual(testTime.minute, 0);
+          });
+          it('Should default the second to 0', function () {
+            strictEqual(testTime.second, 0);
+          });
+          it('should default the millisecond to 0', function () {
+            strictEqual(testTime.millisecond, 0);
+          });
+        });
+        context('With just hour', function () {
+          beforeEach(function () {
+            testTime = PGTime.fromDisplay('12');
+          });
+          it('Should assume meridian is AM', function () {
+            strictEqual(testTime.meridian, MERIDIAN.AM);
+          });
```

Comment:
I went back and forth on that for a while, and ultimately decided that any method that specifically operates on "display" time should _only_ accept 12-hour time, so I think throwing an error on `'13'` is the correct behavior. (Passing in just `'13'` is acceptable with the constructor, e.g. `new PGTime('13');`)

That being said, now I'm kind of thinking that it might make more sense to have the meridian field be required for `fromDisplay`, so passing in `'12'` would also throw an error. What would you think about that?

Reply:
I would say that if there isn’t a way to pass in an afternoon time with just the hour that it would make more sense to require the meridian field for fromDisplay.

Reply:
OK, pushed that change in 3f22f7f1